rust_src/src/windows.rs

```diff
@@ -75,14 +75,6 @@ impl LispWindowRef {
         LispGlyphMatrixRef::new(self.current_matrix)
     }
 
-    pub fn prev(self) -> Self {
-        self.prev.into()
-    }
-
-    pub fn next(self) -> Self {
-        self.next.into()
-    }
```

Comment:
Could these not be preserved but changed to `Option<Self>`?

Reply:
In theory. Or turned into iterators.

But for now, all we need is the EQ check.

Reply:
When dealing with LispObject the right mental model is to treat it like a  Option<LispVal>. Unless the code has already ensured it cannot be nil a LispObject must be treated as None until proven otherwise.